tests/FSharp.Compiler.ComponentTests/EmittedIL/Inlining/Match01.fs.il.debug.bsl

```diff
@@ -21,50 +21,50 @@
                                                                                                       int32) = ( 01 00 02 00 00 00 00 00 00 00 00 00 00 00 00 00 ) 
 
   // --- The following custom attribute is added automatically, do not uncomment -------
-  //  .custom instance void [System.Runtime]System.Diagnostics.DebuggableAttribute::.ctor(valuetype [System.Runtime]System.Diagnostics.DebuggableAttribute/DebuggingModes) = ( 01 00 03 00 00 00 00 00 ) 
+  //  .custom instance void [mscorlib]System.Diagnostics.DebuggableAttribute::.ctor(valuetype [mscorlib]System.Diagnostics.DebuggableAttribute/DebuggingModes) = ( 01 00 01 00 00 00 00 00 ) 
 
   .hash algorithm 0x00008004
   .ver 0:0:0:0
 }
 .mresource public FSharpSignatureData.Match01
 {
-  // Offset: 0x00000000 Length: 0x00000718
+  // Offset: 0x00000000 Length: 0x00000722
   // WARNING: managed resource file FSharpSignatureData.Match01 created
 }
 .mresource public FSharpOptimizationData.Match01
 {
-  // Offset: 0x00000720 Length: 0x000003C3
+  // Offset: 0x00000728 Length: 0x000003B7
   // WARNING: managed resource file FSharpOptimizationData.Match01 created
 }
 .module Match01.exe
-// MVID: {628F4C90-D9D6-6143-A745-0383904C8F62}
+// MVID: {63E6B3F4-8A6A-D886-A745-0383F4B3E663}
 .imagebase 0x00400000
 .file alignment 0x00000200
 .stackreserve 0x00100000
 .subsystem 0x0003       // WINDOWS_CUI
 .corflags 0x00000001    //  ILONLY
-// Image base: 0x0000019CC0BB0000
+// Image base: 0x0000029F19F60000
 
 
 // =============== CLASS MEMBERS DECLARATION ===================
 
 .class public abstract auto ansi sealed Match01
-       extends [System.Runtime]System.Object
+       extends [mscorlib]System.Object
 {
   .custom instance void [FSharp.Core]Microsoft.FSharp.Core.CompilationMappingAttribute::.ctor(valuetype [FSharp.Core]Microsoft.FSharp.Core.SourceConstructFlags) = ( 01 00 07 00 00 00 00 00 ) 
   .class abstract auto autochar serializable nested public beforefieldinit Test1
-         extends [System.Runtime]System.Object
-         implements class [System.Runtime]System.IEquatable`1<class Match01/Test1>,
-                    [System.Runtime]System.Collections.IStructuralEquatable,
-                    class [System.Runtime]System.IComparable`1<class Match01/Test1>,
-                    [System.Runtime]System.IComparable,
-                    [System.Runtime]System.Collections.IStructuralComparable
```

Comment:
baseline unification -  [System.Runtime] and [mscorlib] should likely be [runtime]